code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -102,11 +116,15 @@
 	add_cooldown()
 	addtimer(CALLBACK(src, .proc/sneak_attack_cooldown), HUNTER_POUNCE_SNEAKATTACK_DELAY) //Short delay before we can sneak attack.
 
-/datum/action/xeno_action/stealth/proc/cancel_stealth() //This happens if we take damage, attack, pounce, toggle stealth off, and do other such exciting stealth breaking activities.
+/datum/action/xeno_action/stealth/proc/cancel_stealth(silent = FALSE, change_move_intent = TRUE) //This happens if we take damage, attack, pounce, toggle stealth off, and do other such exciting stealth breaking activities.
 	SIGNAL_HANDLER
 	if(!stealth)//sanity check/safeguard
 		return
-	to_chat(owner, "<span class='xenodanger'>We emerge from the shadows.</span>")
+	if(!silent)
+		to_chat(owner, "<span class='xenodanger'>We emerge from the shadows.</span>")
```

Comment:
Unreachable code

Reply:
Shouldn't be unreachable; silent is by default false. The var exists mainly for the on_upgrade recalibration.